db/agents.lock.yml

```diff
@@ -36,7 +36,11 @@ agents:
   - regex: ^APKXDL.+Android
     type: '36'
     os: '42'
-  - regex: ^Breaker.+CFNetwork
+  - regex: ^Bose/
+    name: '86'
+    type: '37'
+    os: '44'
+  - regex: ^Breaker/iOS
```

Comment:
I no longer see any `CFNetwork` in Breaker user agents:

```
Breaker/iOS (https://breaker.audio)
Breaker/iOS
Breaker/0.7.10 (1034)
Breaker/iOS/0.8.0 (1080)
Breaker/0.7.7 (1003)
Breaker/0.7.5 (947)
```

The top 2 are the iOS ones ... the others are Android.